package.json

```diff
@@ -46,12 +46,14 @@
     "bootstrap": "4.1.2",
     "lodash-fp": "0.10.4",
     "prop-types": "15.6.2",
-    "react": "16.6.0",
+    "react": "^16.6.3",
```

Comment:
I have a question about using `^`. I have preferred to tie the versions exactly to avoid problems with library updates causing conflicts. It happened a lot in the angular version of consent-ui. Is there a reason to use the `compatible-with` syntax instead of the exact version?